cmd/util/util.go

```diff
@@ -0,0 +1,26 @@
+package util
+
+import (
+	"fmt"
+)
+
+func RequiredFlagError(flag string) error {
+	return fmt.Errorf("required flag(s) \"%s\" not set", flag)
+}
+
+// IsNotEmpty is easier to read then len(value) > 0)
+func IsNotEmpty(value string) bool {
+	return len(value) > 0
+}
+
+func IsEmpty(value string) bool {
+	return !IsNotEmpty(value)
+}
+
+func IsRequiredOption(flag string, value string) error {
```

Comment:
I moved this method to `isRequiredOption` in the aws package where it can be called by aws/create.go and aws/destroy.go